cmd/deployment/apm/resync.go

```diff
@@ -28,14 +28,31 @@ import (
 )
 
 var resyncApmCmd = &cobra.Command{
-	Use:     "resync <deployment id>",
+	Use:     "resync [<deployment id> | --all]",
```

Comment:
As per https://github.com/elastic/ecctl/issues/44#issuecomment-562053575
```suggestion
	Use:     "resync {<deployment id> | --all}",
```